jujugui/static/gui/src/app/init/component-renderers-mixin.js

```diff
@@ -435,8 +435,19 @@ const ComponentRenderersMixin = (superclass) => class extends superclass {
       this.state.changeState(storeState);
     };
 
+    const addCharmAnnotations = (entityId) => {
+      const getStartedPath = charmstore._generatePath(
```

Comment:
I guess this is temporary but let's not use "private" methods. Let's add a getStartedURL() method to the charm store client.